test/test_bregman.py

```diff
@@ -352,13 +352,15 @@ def test_sinkhorn2_variants_device_tf(method):
         nx.assert_same_dtype_device(Mb, Gb)
         nx.assert_same_dtype_device(Mb, lossb)
 
+    # Check that everything happens on the GPU
```

Comment:
why did you move this outside of the GPU test?

Reply:
Because Codecov does not test these lines, since it works on cpu, giving less coverage to the overall code. When I leave the device verification to the end, Codecov will give coverage to all lines except the last one. Otherwise, it will not give coverage to most of the lines after the `if` as it was before.

To summarize, it is rather a trick to increase the coverage, which were cases that can still be included, but due to limitations of the Codecov machine, it gives the impression that it worsens the coverage.